nemesis/menu.py

```diff
@@ -1247,3 +1247,55 @@ def str_boost():
     menuGrid.sel_sphere("power", "none")
     menuGrid.use_and_quit()
     memory.main.close_menu()
+
+# ------------------------------
+# Nemesis menus
+
+def arena_purchase_1():
+    memory.main.wait_frames(60)
+    xbox.tap_b()
+    memory.main.wait_frames(15)
+    xbox.tap_b()  # Tidus catcher
+    memory.main.wait_frames(15)
+    xbox.tap_up()
+    xbox.tap_b()  # Confirm
+    memory.main.wait_frames(15)
+    xbox.tap_b()  # Do not equip
+    memory.main.wait_frames(15)
+    xbox.tap_down()
+    xbox.tap_b()  # Yuna catcher
+    memory.main.wait_frames(15)
+    xbox.tap_up()
+    xbox.tap_b()  # Confirm
+    memory.main.wait_frames(15)
+    xbox.tap_up()
+    xbox.tap_b()  # Do equip
+    memory.main.wait_frames(15)
+    xbox.tap_a()
+    memory.main.wait_frames(15)
+    xbox.tap_a()
+    memory.main.wait_frames(15)
+    xbox.tap_up()
+    xbox.tap_a()
+    memory.main.wait_frames(15)
+    xbox.tap_b()
+    memory.main.wait_frames(60)
+
+
+def remove_all_nea():
+    for i in range(7):
+        if memory.main.equipped_armor_has_ability(charNum=i):  # Defaults to NEA
```

Comment:
I would do:

```python

if not memory.main.equipped_armor_has_ability(...):
    continue
```

It'll dedent everything under here. The further indented stuff is, the more cognitive load is required to keep track of all the nesting.